physical/consul.go

```diff
@@ -289,17 +291,59 @@ func setupTLSConfig(conf map[string]string) (*tls.Config, error) {
 	return tlsClientConfig, nil
 }
 
+// Used to run multiple entries via a transaction
+func (c *ConsulBackend) Transaction(txns []TxnEntry) error {
+	if txns == nil {
```

Comment:
Not that it matters a lot, but lately I have been wondering if checking `len(sliceVal)` is better over `sliceVal == nil`, since it can fast path an exit flow.